flink-python/pyflink/proto/flink-fn-execution.proto

```diff
@@ -44,12 +44,33 @@ message UserDefinedFunction {
   // 2. The result of another user-defined function
   // 3. The constant value of the column
   repeated Input inputs = 2;
+
+  // Used in pandas batch over window aggregation
+  int32 windowIndex = 3;
 }
 
 // A list of user-defined functions to be executed in a batch.
 message UserDefinedFunctions {
   repeated UserDefinedFunction udfs = 1;
   bool metric_enabled = 2;
+  repeated Window windows = 3;
+}
+
+// Used to describe the info of over window in pandas batch over window aggregation
+message Window {
+  enum WindowType {
+    RANGE_UNBOUNDED = 0;
+    RANGE_UNBOUNDED_PRECEDING = 1;
+    RANGE_UNBOUNDED_FOLLOWING = 2;
+    RANGE_SLIDING = 3;
+    ROW_UNBOUNDED = 4;
+    ROW_UNBOUNDED_PRECEDING = 5;
+    ROW_UNBOUNDED_FOLLOWING = 6;
+    ROW_SLIDING = 7;
+  }
+  WindowType windowType = 1;
+  int64 lowerBoundary = 2;
```

Comment:
lowerBoundary -> lower_boundary? Same for the other fields. This is to keep consistent for the code style of the proto file.